inertia/__init__.py

```diff
@@ -12,4 +12,5 @@
     "lazy",
     "merge",
     "optional",
+    "always"
```

Comment:
There're trailing commas elsewhere here, so this should have one too:
```suggestion
    "always",
```